modules/apps/27-interchain-accounts/host/client/cli/tx_test.go

```diff
@@ -137,6 +141,7 @@ func TestGeneratePacketData(t *testing.T) {
 }
 
 func assertMsgBankSend(t *testing.T, msg sdk.Msg) {
+	t.Helper()
```

Comment:
I'm unsure if this should actually be flagged a helper function, this is an assertion function, with the same purpose as something like `require.True(t, ok)`. What is the benefit of marking this as a helper? (same applies to the functions that begin with `assert`)

Reply:
Am I understanding this correctly that `t.Helper` will point up the call stack instead of the specific line the error occurs at? The idea being multiple functions call the helper?

Reply:
I think we would want the test to fail at a line in the assertionFn though right, .e.g. I would want to know that this line failed

`require.Equal(t, "cosmosvaloper1qnk2n4nlkpw9xfqntladh74w6ujtulwnmxnh3k", msgDelegate.ValidatorAddress)`

rather than

`tc.assertMsgDelegate(t, msg)`

unless I am misunderstanding something

Reply:
Am understanding the same thing so I'm now less inclined to believe I'm not understanding something.

+1 on wanting the error trace to point at most relevant spot.

Reply:
how abvout the assertMsgDelegate below?

Should that have it?

Reply:
So my initial thought was that it the same logic should apply, however I just ran some tests that I modified to fail, and the stack trace points exactly where I would expect with and without the `t.Helper()` call. (at the failing assertion within the `assertMsgDelegate` )

I'm not actually noticing a difference in the output.